californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfigDefaults.java

```diff
@@ -76,7 +76,7 @@ public static void setDefaults(final NetworkConfig config) {
 		config.setInt(NetworkConfig.Keys.PREFERRED_BLOCK_SIZE, 512);
 		config.setInt(NetworkConfig.Keys.MAX_MESSAGE_SIZE, 1024);
 		config.setInt(NetworkConfig.Keys.MAX_RESOURCE_BODY_SIZE, DEFAULT_MAX_RESOURCE_BODY_SIZE);
-		config.setInt(NetworkConfig.Keys.BLOCKWISE_STATUS_LIFETIME, 10 * 60 * 1000); // ms
+		config.setInt(NetworkConfig.Keys.BLOCKWISE_STATUS_LIFETIME, DEFAULT_BLOCKWISE_STATUS_LIFETIME); // ms
```

Comment:
i was a bit confuse when I saw this 30 seconds too.
Is it the whole time available to get or send the whole payload ? I mean all the blockwise request/response exchanges needed to send or receive the initial message payload ? Even 5 minutes seems to me really short. Just to get the first ACK of the first request this can take 247s (EXCHANGE_LIFETIME)